### ISSUE TITLE
Demo is not working

### ISSUE THREAD

**Issue by Tyki** [Atyantik/react-pwa#108]
https://demo.reactpwa.com/ leads to a 502 error

**Comment by tirthbodawala** [comment]
@Tyki yes we are working on the demo at the moment. we are coming with not just a hello world program but more than that to explain what ReactPWA can do.

![image](https://user-images.githubusercontent.com/11286271/47644909-74251f00-db95-11e8-8722-8ae3ca3ee51d.png)
